cassandra-migration/src/main/resources/db/migration/cassandra/V1.27_cassandra.cql

```diff
@@ -0,0 +1 @@
+ALTER TABLE bulk_upload_process ALTER uploadeddate TYPE timestamp;
```

Comment:
The expected columns are. Update accordingly.
createdby 
createdon (timestamp)
lastupdatedon (timestamp)